js/Readium.js

```diff
@@ -142,7 +142,13 @@ define(['readium_shared_js/globals', 'text!version.json', 'jquery', 'underscore'
 
 
         this.openPackageDocument = function(ebookURL, callback, openPageRequest)  {
-                        
+            var resourcePluginKey='ResourceFetcherPlugin';
+            var plugins = ReadiumSDK.Plugins.getLoadedPlugins()
+            if (_.has(plugins, resourcePluginKey)) {
```

Comment:
Another way to test for whether or not a plugin is loaded, based on its registered name:
https://github.com/readium/readium-js/blob/develop/dev/index.js#L146
```javascript
if (ReadiumSDK.reader.plugins.ResourceFetcherPlugin) {
...
}
```
(I'm not 100% sure that `ReadiumSDK.reader` is valid ... to be verified)